src/AdminConsole/Pages/App/Settings/Settings.cshtml.cs

```diff
@@ -206,7 +206,9 @@ public async Task<IActionResult> OnPostSettingsAsync()
 
         try
         {
-            if (Convert.ToBoolean(Request.Form[ManualVerificationTokenCheckboxId].FirstOrDefault()))
+            Request.Form.TryGetValue(ManualVerificationTokenCheckboxId, out var values);
```

Comment:
Why is it not getting it from the form's model directly/

Reply:
For some reason the bindings aren't working on that page. Do you see something that's missing from the previous version of the page?

Reply:
Fixed!